src/rulesets/SparseArrays/array.jl

```diff
@@ -0,0 +1,16 @@
+using SparseArrays
+
+function rrule(::Type{<:SparseMatrixCSC{T,N}}, arr) where {T,N}
```

Comment:
we try to match the file structure of Base / stdlibs (althought we're not yet entire consistent on that). Since `SparseMatrixCSC` is defined in `SparseArrays/src/sparsematrix.jl`, can you please rename this file from `rulesets/SparseArrays/array.jl` -> `rulesets/SparseArrays/sparsematrix.jl`

Reply:
Done